HPXMLtoOpenStudio/resources/unit_conversions.rb

```diff
@@ -17,6 +17,8 @@ class UnitConversions
     ['gj', 'kwh'] => 277.778,
     ['gj', 'therm'] => 9.48043,
     ['kwh', 'btu'] => 3412.141633127942,
+    ['kwh', 'kbtu'] => 3412.141633127942 / 1000.0,
+    ['kwh', 'mbtu'] => 3412.141633127942 / 1000.0 / 1000.0,
```

Comment:
These aren't needed, we do these conversions all the time. The code is smart enough to know that `[a, b] => X` is the same as `[b, a] => 1/X`.